app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -2,6 +2,10 @@ module Idv
   module VerifyInfoConcern
     extend ActiveSupport::Concern
 
+    def capture_secondary_id_enabled
+      IdentityConfig.store.in_person_capture_secondary_id_enabled
```

Comment:
Does this need to be in VerifyInfoConcern when it's not being used by the remote verify_info_controller? If you keep this in the in_person verify_info_controller, then it's really clear that the remote version is not being affected because none of the common code is being touched.

Reply:
Good suggestion. I moved it out.  /verify_info_concenr.rb is no longer in Files changed list